portal-web/test/functional/com/liferay/portalweb/tests/MyDMTests/MyDMTests.testcase

```diff
@@ -0,0 +1,192 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "Can Create a Documents and Media"
+	@priority = 5
+	test CanCreateDM {
+		task ("Given the DM Portlet is added to the page") {
+			Navigator.openURL();
```

Comment:
You don't need this function here, since is used on the setUp. This kind of functions does not affect the test, but the idea is to be more optimal as possible, to save even save CI resources if we look into scale.